src/main/scala/com/kinja/amqp/persistence/AwsAZMessageStore.scala

```diff
@@ -1,7 +1,7 @@
 package com.kinja.amqp.persistence
 
 /**
-  * Interface for marking message stores which persist data only in one AWS availability zone.
-  * (Data available only for one cluster in our case.)
-  */
+ * Interface for marking message stores which persist data only in one AWS availability zone.
+ * (Data available only for one cluster in our case.)
+ */
```

Comment:
Huh? I thought the old formatting was correct.

Reply:
The spaces and the tabs...  Probably last time I've forgot to call the compiler before the commit when I've add this doc.